pkg/apiserver/rest/dualwriter.go

```diff
@@ -143,75 +143,6 @@ type NamespacedKVStore interface {
 	Set(ctx context.Context, key, value string) error
 }
 
-func SetDualWritingMode(
```

Comment:
I don't think this is doing anything anymore -- but correct me if I am wrong.  Involving the database in this configuration seems overwhelming 😬

Reply:
In fact, I think that we need this because it's the method that runs all validation/gates before switching from one mode to another. Also, it keeps track of which mode was running on a particular instance at each time.

Reply:
c/c @suntala  @leonorfmartins